datahub-classify/src/datahub_classify/infotype_predictor.py

```diff
@@ -25,19 +25,28 @@ def get_infotype_function_mapping(
         else:
             fn_name = f"inspect_for_{infotype.lower()}"
             infotype_function_map[infotype] = module_fn_dict[fn_name]
-    return infotype_function_map
+    return infotype_function_map#,  spacy_models_list
 
 
 def predict_infotypes(
     column_infos: List[ColumnInfo],
     confidence_level_threshold: float,
+    language: str,
     global_config: Dict[str, Dict],
     infotypes: Optional[List[str]] = None,
 ) -> List[ColumnInfo]:
+    
     infotype_function_map = get_infotype_function_mapping(infotypes, global_config)
     logger.debug(f"Total columns to be processed --> {len(column_infos)}")
     logger.debug(f"Confidence Level Threshold set to --> {confidence_level_threshold}")
+    logger.debug(f"Spacy language set to --> {language}")
     logger.debug("===========================================================")
+
+    module_name = "datahub_classify.infotype_helper"
+    function_name ="init_spacy"
+    module = importlib.import_module(module_name)
+    init_spacy = getattr(module, function_name)
+    spacy_models_list = init_spacy(language)
```

Comment:
Maintain the “spacy_models_list” at global level and initialize it in the first API call as per the user specified language, this will avoid repetitive loading of the spacy model. 
In the subsequent call, validate the loaded model and proceed further (for this, you may have a dict (spacy_model_dict) with key as language name and value as respective spacy model, if language is present in dict then no need to load it).